metadata-load/src/test/resources/application.properties

```diff
@@ -20,3 +20,4 @@ spring.jmx.enabled=false
 entrez.api.key=
 entrez.api.attempts=10
 entrez.api.delay=1000
+liquibase.enabled=false
```

Comment:
is it mandatory in metadata-load as well

Reply:
Verified removing this, test fails

Reply:
Sure, we don't want to enable Liquibase in the test environment, only in the main one. Marking this as resolved.